common/scala/src/main/scala/whisk/core/WhiskConfig.scala

```diff
@@ -83,6 +84,8 @@ class WhiskConfig(
     val mainDockerEndpoint = this(WhiskConfig.mainDockerEndpoint)
     val routerDockerEndpoint = this(WhiskConfig.routerDockerEndpoint)
     val elkDockerEndpoint = this(WhiskConfig.elkDockerEndpoint)
+
+    lazy val logSizeLimit = ObjectSize.fromString(this(WhiskConfig.logSizeLimit))
```

Comment:
If the value is not set (and not even required by the class instantiating WhiskConfig) an expection is thrown by the `fromString` method.